static/css/com/layout/header.scss

```diff
@@ -9,6 +9,15 @@
   background-color: #27282c;
   background-size: cover;
   min-width: 960px;
+  position: relative;
+  display: flex;
```

Comment:
I'm not sure, why are we use `flex` for `global-header`? Cause, it's has just one child with full size.

Reply:
It's redundant, I've removed it, thanks)